src/queues/job-queue.js

```diff
@@ -78,6 +80,12 @@ const initQueue = () => {
   registerV2Queue.on('completed', handlerCompleted);
   registerV2Queue.on('stalled', handlerStalled);
 
+  console.log('Follow topic job is working');
+  followTopicQueue.process(followTopicProcess);
+  followTopicQueue.on('failed', handlerFailure);
+  followTopicQueue.on('completed', handlerCompleted);
+  followTopicQueue.on('stalled', handlerStalled);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While the code itself looks fine, it's important to note that logging "Follow topic job is working" before actually starting the process might be misleading. If the `followTopicQueue.process(followTopicProcess);` line throws an error or fails for some reason, the log message would have already been printed, potentially causing confusion during debugging. Consider moving the log statement after the process initiation.
```diff
-   console.log('Follow topic job is working');
    followTopicQueue.process(followTopicProcess);
+   console.log('Follow topic job is working');
    followTopicQueue.on('failed', handlerFailure);
    followTopicQueue.on('completed', handlerCompleted);
    followTopicQueue.on('stalled', handlerStalled);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->